cartography/intel/gcp/storage.py

```diff
@@ -0,0 +1,223 @@
+import logging
+from googleapiclient.discovery import HttpError
+
+from cartography.intel.gcp import compute
+logger = logging.getLogger(__name__)
+
+
+def get_gcp_bucket_metadata(storage, bucket):
```

Comment:
I think this is super helpful but not a must for a first PR.  Let's just get the buckets first and then drill down in a second PR.

Reply:
Will do!